### DISCUSSION TITLE
Sign up email sent from Supabase not the one in authentication section

### DISCUSSION THREAD

**Discussion by udrscr** [supabase/supabase#29366] [Questions]
Hello all! 👋

Does anyone knows why email that I have under "Email templates" inside "Authentication" section is not the one that is sent by Supabase when new user is signed up?

The code base was not changes in the last month and everything was working fine couple days ago when I checked last time.

Was something updated regarding the auth in Supabase in the last two weeks?

**Comment by chuckcg** [comment]
I have a similar problem. The emails in Email templates aren't the ones being sent. Instead, it seems to be the default emails templates that are sent. 

Works perfectly locally (by updating config.toml and using a html file as template), doesn't work live (via the dashboard ui)

**Reply by GaryAustin1** [reply]
Not seeing others with issues anywhere here or Discord.  I scanned recent PR's and don't see a change.

Are you sure you are changing the correct template and on the correct instance (running locally, another instance)?

Hard reset your instance and hard refresh your browser.

Otherwise unless someone else comes along report to Support.

**Comment by chuckcg** [comment]
Yeah I triple checked that it was the right instance. 

I also already tried to restart it. And tried on a fresh browser. 

I just found something out: When I send the password request from supabase itself (Authentication > Users > Send password recovery), it does work ! It picks up the right template.

So maybe it's an issue with the [way I ask for a password reset](https://github.com/orgs/supabase/discussions/29976#discussion-7336089) ? I can't find much else in the docs...
